### PR TITLE
[#2092] doesn't try to validate if value is blank

### PR DESCRIPTION
**Pull request by jsadler2**
@pkdash could you please review this small PR.  Now when a new version of a SWAT Model Instance is created, it does not try to validate the value, because there is no value to validate.

### CODE REVIEW

File: hs_swat_modelinstance/models.py [hunk 1]
```diff
@@ -333,18 +333,19 @@ def _validate_time_step(cls, **kwargs):
             if val == 'Choose a type':
                 kwargs[key] = ''
             else:
```
Comment (pkdash) [inline hs_swat_modelinstance/models.py]: ```else``` -> ```elif val != '':```

File: hs_swat_modelinstance/models.py [hunk 2]
```diff
@@ -333,18 +333,19 @@ def _validate_time_step(cls, **kwargs):
...
-                    time_step = val
-                    types = [c[0] for c in cls.routing_type_choices]
-                    cls.check_time_steps(time_step, types)
-                elif key == 'simulationTimeStepType':
-                    time_step = val
-                    types = [c[0] for c in cls.simulation_type_choices]
-                    cls.check_time_steps(time_step, types)
+                if val != '':
+                    if key == 'rainfallTimeStepType':
+                        time_step = val
```
Comment (pkdash) [inline hs_swat_modelinstance/models.py]: This code line ```time_step = val``` should go above the ```if .. else``` block so that it is set only once.

**Comment by jsadler2** [comment]
@pkdash thanks for those comments. I have made those changes now. Were there any other changes you'd like us to make?

**Comment by pkdash** [comment]
@jsadler2 Looks good. +1. Merge after Jenkins checks are successful.

**Comment by aphelionz** [comment]
@jsadler2 Is this related to an explicit issue?

**Comment by jsadler2** [comment]
@aphelionz yes. #2092

**Pull request merged**